Source/ASNetworkImageNode.h

```diff
@@ -196,6 +196,38 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)imageNodeDidFinishDecoding:(ASNetworkImageNode *)imageNode;
 
+/**
+ * Notification that the image node finished downloading an image, with additional info.
+ * If implemented, this method will be called instead of `imageNode:didFetchImage:`.
+ *
+ * @param imageNode The sender.
+ * @param image The newly-loaded image.
+ * @param info Additional information about the image load.
+ *
+ * @discussion Called on a background queue.
+ */
+- (void)imageNode:(ASNetworkImageNode *)imageNode didFetchImage:(UIImage *)image info:(ASNetworkImageLoadInfo *)info;
+
+/**
+ * Notification that the image node finished downloading an image.
+ *
+ * @param imageNode The sender.
+ * @param image The newly-loaded image.
+ *
+ * @discussion Called on a background queue.
+ */
+- (void)imageNode:(ASNetworkImageNode *)imageNode didFetchImage:(UIImage *)image;
```

Comment:
Creating `ASNetworkImageLoadInfo` objects is really cheap, so if we're revising this API I'd advocate for removing this method and only providing `didFetchImage:info:` and `didFailFetchingWithError:`.

If you choose to go that route, update the docs for `didFetchImage:info:` to remove the reference to this method.